newsletter-april-2021.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: April 5, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from April.
+
+Read the Docs newsletter - April 2021
+=====================================
+
+This is the first of our monthly newsletters, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
+
+Company highlights
+------------------
+
+-  We have a new colleague! `Juan Luis <https://blog.readthedocs.com/archive/author/juan-luis-cano-rodriguez/>`_
+   will be working with us as Developer Advocate, with a focus on fulfilling
+   the goals of `the CZI grant we were
+   awarded <https://blog.readthedocs.com/czi-grant-announcement/>`_,
+   improve our public facing documentation,
+   and spread the word about our service.
+-  xxx we saw an increment of X % page views
+-  We have `a new status page <http://status.readthedocs.com/>`_ so you
+   can see the availability history and current status of our infrastructure.
+
+New features
+------------
+
+-  There is a `new support form <https://docs.readthedocs.io/en/latest/support.html>`_
+   in case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`_
+   if you find any problems with it.
+-  Now `all newly created MkDocs projects use the latest version of mkdocs by
+   default <https://github.com/readthedocs/readthedocs.org/pull/7869>`_.
+   Existing projects can specify a specific version, but won't automatically
+   be upgraded.
+-  We expanded a few areas of our user documentation: we added `a brand
+   new guide on reproducible
+   builds <https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html>`_,
+   considerably improved our `guide on using
+   conda <https://docs.readthedocs.io/en/stable/guides/conda.html>`_,
+   and `started recommending MyST for writing Markdown with
+   Sphinx <https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx>`_.
```

Comment:
Think https://github.com/readthedocs/readthedocs.org/pull/7891 is a good new improvement to mention here :)

Reply:
Good one! Done :shipit: